examples/quilkin-filter-example/Cargo.toml

```diff
@@ -22,13 +22,17 @@ repository = "https://github.com/googleforgames/quilkin"
 edition = "2018"
 
 [dependencies]
-quilkin = "0.1.0"
-tokio = { version = "1", features = ["full"]}
-prost = "0.7"
-prost-types = "0.7"
+# If lifting this example, you will want to be explicit abou the Quilkin version, e.g.
+# quilkin = "0.2.0"
+quilkin = { path = "../../" }
```

Comment:
This was the best way I could think of to ensure that the code stayed in sync. Is there a better way?

Reply:
I think this is reasonable. This code is never being published to crates.io, so using paths is totally fine.